### PR TITLE
fix(content-explorer): Fix color contrast issues

### PR DESCRIPTION
**Pull request by tjuanitas**
The following changes have been consulted with design:

1. Increase icon color of Grid/List View button (`ViewModeChangeButton`). This aligns with color used in main web app.

**Before**
Screen Shot 2023-01-25 at 10 03 45 AM
**After**
Screen Shot 2023-01-25 at 10 04 40 AM

~~2. Improve contrast on "More Options" button by adding white background to button and darker icon color. This is a similar pattern from Admin Console~~

~~**Before**
Screen Shot 2023-01-25 at 10 06 24 AM
**After**
Screen Shot 2023-01-25 at 10 06 43 AM~~

3. Change color of header columns and add focus outline. Closer aligns to main web app

**Before**
Screen Shot 2023-01-25 at 10 09 30 AM
**After**
Screen Shot 2023-01-25 at 10 09 45 AM

---

Edit:

- Updated "Sort" and "Add" icon colors to match "Grid/List" icons

**Before**
Screen Shot 2023-01-26 at 7 58 32 PM
**After**
Screen Shot 2023-01-26 at 7 56 41 PM

- Updated border and icon color for "More Options" button states

**Hover State - Before**
Screen Shot 2023-01-26 at 8 03 32 PM

**Hover State - After**
Screen Shot 2023-01-26 at 8 04 27 PM

**Focus State - Before**
Screen Shot 2023-01-26 at 8 03 47 PM

**Focus State - After**
Screen Shot 2023-01-26 at 8 04 35 PM

### CODE REVIEW

**Review by jstoffan — COMMENTED** [review]
@tjuanitas, do the border colors need to be updated, as well? They seem to be the same color as the icons in the current design.

**Comment by tjuanitas** [comment]
@jstoffan I'll double check all the colors with my designer. The `ViewModeChangeButton` border seems okay to me because the web app also uses the default border-color for icon buttons but the "More Options" button seems off.

File: src/elements/content-explorer/ItemList.scss [hunk 1]
```diff
@@ -56,6 +57,10 @@
             opacity: 1;
         }
 
+        .bce-btn-more-options {
+            border-color: $grey-blue;
```
Comment (tjuanitas) [inline src/elements/content-explorer/ItemList.scss:61]: using a deprecated color to match current item row border-color

File: src/elements/common/sub-header/AddButton.js [hunk 2]
```diff
@@ -10,12 +10,13 @@ import Button from '../../../components/button';
...
 import messages from '../messages';
 import Tooltip from '../Tooltip';
+import { bdlGray62 } from '../../../styles/variables';
 import './AddButton.scss';
 
 const AddButton = (props: ?Object) => (
     <Tooltip text={<FormattedMessage {...messages.add} />}>
         <Button className="be-btn-add" aria-label={messages.add.defaultMessage} type="button" {...props}>
-            <IconAddThin />
+            <IconAddThin color={bdlGray62} />
```
Comment (tjuanitas) [inline src/elements/common/sub-header/AddButton.js:19]: legacy icon colors are modified through props

**Comment by tjuanitas** [comment]
@jstoffan We ended up making a couple more changes to align the color of the icons on the sub-header component of Explorer and Picker, and also updated the border color of the "More Options" button. Added more screenshots to the description.

**Review by QingyuChai — APPROVED** [review]

**Review by jstoffan — APPROVED** [review]

**Pull request merged**